### DISCUSSION TITLE
Problem with IPageDialogService and INavigationService

### DISCUSSION THREAD

**Discussion by MichalPeterek** [PrismLibrary/Prism#3099] [.NET MAUI]
Hi,
After updating MS Visual studio ( and MAUI ) I receive this exceptions :

`DryIoc.ContainerException
  Message=code: Error.UnableToResolveUnknownService;
message: Unable to resolve Prism.Services.IPageDialogService as parameter "Dialogs"
  in Scoped XXX.ViewModels.BaseServices as parameter "baseServices" FactoryId=877
  in resolution root Scoped XXX.ViewModels.ProPlayerViewModel FactoryId=818 (TracksTransientDisposable, IsResolutionCall)
  from container without scope
 with Rules with {TrackingDisposableTransients, FuncAndLazyWithoutRegistration, SelectLastRegisteredFactory, UsedForValidation} and without {ThrowOnRegisteringDisposableTransient, ImplicitCheckForReuseMatchingScope, EagerCachingSingletonForFasterAccess} with TotalDependencyCountInLambdaToSplitBigObjectGraph=2147483647
 with FactorySelector=SelectLastRegisteredFactory
 with Made={FactoryMethod=ConstructorWithResolvableArguments}
Where no service registrations found
  and no dynamic registrations found in 1 of Rules.DynamicServiceProviders
  and nothing found in 0 of Rules.UnknownServiceResolvers`

How to I can make it work ? 

Edit:

Only happens when starting by debugger. Issue is not occurs while starting deployed app manual from phone.

**Comment by dansiegel** [comment]
It's hard to say what your issue is without a reproduction. It works fine in the e2e app in the repo.

**Comment by DonkeyKongJr** [comment]
@MichalPeterek have you ever found the resaon for this?

**Reply by MichalPeterek** [reply]
Yes deleting obj and bin folder and call dotnet restore solved this.

**Reply by munkii** [reply]
@DonkeyKongJr do you still get this error?
